src/main/java/zuve/searchablechests/RichTextFieldWidget.java

```diff
@@ -1,722 +1,60 @@
 package zuve.searchablechests;
```

Comment:
Thank you for changing this to be a subclass, something I've been meaning to do!